src/adapters/poi/map_poi.js

```diff
@@ -1,8 +1,9 @@
-import Poi from "./poi";
+import Poi,{POI_TYPE} from "./poi";
 
 export default class MapPoi extends Poi {
   constructor(feature, lngLat) {
     let id = feature.properties.global_id
-    super(id,  feature.name, 'poi', lngLat)
+    console.log(feature)
+    super(id,  feature.name,'to replace', POI_TYPE, lngLat)
```

Comment:
`'to_replace'` : is it temporary ? Or should it be null ?

Reply:
was temporary until i don't find matching data to fit alternative name (fixed to null)